starlark/benches/rust-benches/naive_sort.sky

```diff
@@ -0,0 +1,10 @@
+def native_sort(array):
```

Comment:
Isn't that bubblesort?

Reply:
It is not exactly bubble sort. But I have rewritten it to be bubble sort.